assets/templates/kubernetes.container/fields.yml

```diff
@@ -0,0 +1,42 @@
+- name: timestamp
+  type: date 
+- name: timedate
+  type: keyword   
+- name: timehour
+  type: keyword   
+- name: agent.id
+  type: keyword
+- name: agent.name
+  type: keyword
+- name: agent.version
+  type: keyword
+- name: agent.ephemeral_id
+  type: keyword  
+- name: container.network.ingress.bytes
```

Comment:
any reason for having this as `double` and then casting to `int` in the template?
same for `container.network.egress.bytes`, `Offset` and `faults`

Reply:
Fixed in 31b2839e62064fc4d20c24fd78079e161efa8d06